spec/requests/v1/items_request_spec.rb

```diff
@@ -101,7 +101,7 @@
         delete "/api/v1/items/#{item.id}"
     
         expect(response).to be_successful
-        expect(Item.count).to eq(10)
-        expect{Item.find(poster.id) }.to raise_error(ActiveRecord::RecordNotFound)
+        expect(Item.count).to eq(4)
```

Comment:
Scratch my previous comment. This was what I initially expected this value to become, good catch :)